java/core/src/main/java/com/microsoft/bond/StructBondType.java

```diff
@@ -2053,6 +2197,10 @@ public final void serialize(
             this.verifyFieldWasNotYetDeserialized(wasAlreadyDeserialized);
             return this.fieldType.deserializeSomethingField(context, this);
         }
+
+        public final SomethingObject<String> deserialize(UntaggedDeserializationContext context) throws IOException {
+            return SomethingObject.wrap(this.fieldType.deserializeValue(context));
```

Comment:
>SomethingObject.wrap [](start = 19, length = 20)

SomethingObject inherits wrap methods from Something class. I suggest using the actual declaring class when you call a static method. 

Is there a reason you didn't rely on the import of the "wrap" methods and need to add the class name?